src/cljs/nnts2/organization/api.cljs

```diff
@@ -0,0 +1,16 @@
+(ns nnts2.organization.api
+  (:require [ajax.core :as ajax]
+            [re-frame.core :as re-frame]
+            [nnts2.db :as db]))
+
+(enable-console-print!)
+
+(defn create-map
```

Comment:
naming is not correct here

Reply:
This function returns a map that contains all the http request information (in a map) for sending a POST request to create an org. Hence the name create-map.